EmotiBit.cpp

```diff
@@ -3728,6 +3747,34 @@ void EmotiBit::processFactoryTestMessages()
 		{
 			hibernate();
 		}
+		else if (msgTypeTag.equals(EmotiBitPacket::TypeTag::SERIAL_DATA_ON))
+		{
+			String dataType;
+			EmotiBitPacket::getPacketElement(msg, dataType, 3);
```

Comment:
Maybe we could change the hardcoded `3` in the function call? but i dont think its blocking